pageserver/src/walredo.rs

```diff
@@ -626,35 +592,37 @@ impl PostgresRedoProcess {
 
         // Create empty data directory for wal-redo postgres, deleting old one first.
         if datadir.exists() {
-            info!(
-                "old temporary datadir {} exists, removing",
-                datadir.display()
-            );
+            info!("old temporary datadir {datadir:?} exists, removing",);
             fs::remove_dir_all(&datadir)?;
         }
         let pg_bin_dir_path = conf.pg_bin_dir(pg_version).map_err(|e| {
             Error::new(
                 ErrorKind::Other,
-                format!("incorrect pg_bin_dir path: {}", e),
+                format!("incorrect pg_bin_dir path: {e:#}"),
             )
         })?;
         let pg_lib_dir_path = conf.pg_lib_dir(pg_version).map_err(|e| {
             Error::new(
                 ErrorKind::Other,
-                format!("incorrect pg_lib_dir path: {}", e),
+                format!("incorrect pg_lib_dir path: {e:#}"),
             )
         })?;
 
-        info!("running initdb in {}", datadir.display());
-        let initdb = Command::new(pg_bin_dir_path.join("initdb"))
-            .args(["-D", &datadir.to_string_lossy()])
-            .arg("-N")
-            .env_clear()
-            .env("LD_LIBRARY_PATH", &pg_lib_dir_path)
-            .env("DYLD_LIBRARY_PATH", &pg_lib_dir_path) // macOS
-            .close_fds()
-            .output()
-            .map_err(|e| Error::new(e.kind(), format!("failed to execute initdb: {e}")))?;
+        info!("running initdb in {datadir:?}");
```

Comment:
All callers of prepare_initdb_command to similar kind of logging.
Can we centralize this inside `prepare_initdb_command`?
Ofcourse, that woudl require `RUST_LOG` to include a rule for `postgres_ffi`.

Alternatively, introduce an `InitdbArgs` struct, make it `derive(Debug)`, and log `info!("running initdb: {args:?}")` at the caller.

Reply:
Sure, `postgres_ffi` has `log` crate and we use adapters for that, added.